tests/test_databases.py

```diff
@@ -87,6 +87,8 @@ def create_test_database():
             url = str(database_url.replace(driver="pymysql"))
         elif database_url.scheme == "postgresql+aiopg":
```

Comment:
I think it should be something like this:

```python
if database_url.scheme in ["postgresql+asyncpg", "postgresql+aiopg"]:
    ...
if database_url.scheme in ["mysql", "mysql+aiomysql"]:
    ...
if database_url.scheme in ["aiosqlite"]:
    ....
```

We should also make this work for  `mysql+aiomysql` and `sqlite+aiosqlite` to be consistent.